example/lib/pages/animated_map_controller.dart

```diff
@@ -34,23 +35,31 @@ class AnimatedMapControllerPageState extends State<AnimatedMapControllerPage> wi
     mapController = new MapController();
   }
 
-  void _animatedMapMove (LatLng destLocation, double destZoom) {
+  void _animatedMapMove(LatLng destLocation, double destZoom) {
     // Create some tweens. These serve to split up the transition from one location to another.
     // In our case, we want to split the transition be<tween> our current map center and the destination.
-    final _latTween = new Tween<double>(begin: mapController.center.latitude, end: destLocation.latitude);
-    final _lngTween = new Tween<double>(begin: mapController.center.longitude, end: destLocation.longitude);
-    final _zoomTween = new Tween<double>(begin: mapController.zoom, end: destZoom);
+    final _latTween = new Tween<double>(
+        begin: mapController.center.latitude, end: destLocation.latitude);
+    final _lngTween = new Tween<double>(
+        begin: mapController.center.longitude, end: destLocation.longitude);
+    final _zoomTween =
+        new Tween<double>(begin: mapController.zoom, end: destZoom);
 
     // Create a new animation controller that has a duration and a TickerProvider.
-    AnimationController controller = AnimationController(duration: const Duration(milliseconds: 500), vsync: this);
+    AnimationController controller = AnimationController(
+        duration: const Duration(milliseconds: 500), vsync: this);
     // The animation determines what path the animation will take. You can try different Curves values, although I found
     // fastOutSlowIn to be my favorite.
-    Animation<double> animation =  CurvedAnimation(parent: controller, curve: Curves.fastOutSlowIn);
+    Animation<double> animation =
+        CurvedAnimation(parent: controller, curve: Curves.fastOutSlowIn);
 
     controller.addListener(() {
       // Note that the mapController.move doesn't seem to like the zoom animation. This may be a bug in flutter_map.
-      mapController.move(LatLng(_latTween.evaluate(animation), _lngTween.evaluate(animation)), _zoomTween.evaluate(animation));
-      print("Location (${_latTween.evaluate(animation)} , ${_lngTween.evaluate(animation)}) @ zoom ${_zoomTween.evaluate(animation)}");
+      mapController.move(
+          LatLng(_latTween.evaluate(animation), _lngTween.evaluate(animation)),
+          _zoomTween.evaluate(animation));
+      print(
+          "Location (${_latTween.evaluate(animation)} , ${_lngTween.evaluate(animation)}) @ zoom ${_zoomTween.evaluate(animation)}");
```

Comment:
This is left over from another contribution, but in any case we should remove it